package.json

```diff
@@ -37,7 +37,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!axios)/\"",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint 'src/*.{js,ts,tsx}'",
+    "lint:fix": "eslint 'src/*.{js,ts,tsx}' --fix",
+    "prettier": "prettier --write 'src/*.{js,ts,tsx}' 'src/*.{scss,css}'"
```

Comment:
I think, need to add precommit script in this PR
 "precommit": "npm run lint:fix && npm run prettier"

Reply:
I icheck in my pc and scripts work like this format:

    "lint": "eslint src/**/*.{js,ts,tsx}",
    "lint:fix": "eslint src/**/*.{js,ts,tsx} --fix",
    "prettier": "prettier --write 'src/**/*.{js,ts,tsx}' 'src/**/*.{scss,css}'",